src/resources/postcss/components/_small-cta.pcss

```diff
@@ -6,13 +6,28 @@
 
 .tribe-events {
 
+	.tribe-events-c-small-cta {
+
+		> * {
+			margin-left: 10px;
+
+			&:first-child {
+				margin-left: 0;
+			}
+		}
+	}
+
 	.tribe-events-c-small-cta__link {
 		cursor: pointer;
 	}
 
-	.tribe-events-c-small-cta__price {
+	.tribe-events-c-small-cta__price,
+	.tribe-events-c-small-cta__stock {
 		@mixin desktop-body-3;
```

Comment:
you have this defined already on the markup using the `tribe-common-b3` class, we don't need this.